[2023-03-03T07:44:55.727+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.scrape_nexrad_data manual__2023-03-03T07:44:47.596700+00:00 [queued]>
[2023-03-03T07:44:55.738+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.scrape_nexrad_data manual__2023-03-03T07:44:47.596700+00:00 [queued]>
[2023-03-03T07:44:55.740+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-03T07:44:55.744+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-03T07:44:55.749+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-03T07:44:55.770+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): scrape_nexrad_data> on 2023-03-03 07:44:47.596700+00:00
[2023-03-03T07:44:55.789+0000] {standard_task_runner.py:55} INFO - Started process 126 to run task
[2023-03-03T07:44:55.798+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'metadata_scraper', 'scrape_nexrad_data', 'manual__2023-03-03T07:44:47.596700+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/scrapper.py', '--cfg-path', '/tmp/tmpsty85lqj']
[2023-03-03T07:44:55.799+0000] {standard_task_runner.py:83} INFO - Job 55: Subtask scrape_nexrad_data
[2023-03-03T07:44:55.901+0000] {task_command.py:388} INFO - Running <TaskInstance: metadata_scraper.scrape_nexrad_data manual__2023-03-03T07:44:47.596700+00:00 [running]> on host 951056ee0f21
[2023-03-03T07:44:55.967+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Charu Singh
AIRFLOW_CTX_DAG_ID=metadata_scraper
AIRFLOW_CTX_TASK_ID=scrape_nexrad_data
AIRFLOW_CTX_EXECUTION_DATE=2023-03-03T07:44:47.596700+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-03T07:44:47.596700+00:00
[2023-03-03T07:44:55.977+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scrapper.py", line 112, in NEXRAD_scrap_load
    result = s3.list_objects(Bucket=nexrad_bucket, Prefix=prefix, Delimiter='/')
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/client.py", line 530, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/client.py", line 944, in _make_api_call
    operation_model, request_dict, request_context
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/client.py", line 966, in _make_request
    return self._endpoint.make_request(operation_model, request_dict)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/endpoint.py", line 119, in make_request
    return self._send_request(request_dict, operation_model)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/endpoint.py", line 198, in _send_request
    request = self.create_request(request_dict, operation_model)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/endpoint.py", line 137, in create_request
    operation_name=operation_model.name,
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/hooks.py", line 412, in emit
    return self._emitter.emit(aliased_event_name, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/hooks.py", line 256, in emit
    return self._emit(event_name, kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/hooks.py", line 239, in _emit
    response = handler(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/signers.py", line 105, in handler
    return self.sign(operation_name, request)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/signers.py", line 189, in sign
    auth.add_auth(request)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/auth.py", line 418, in add_auth
    raise NoCredentialsError()
botocore.exceptions.NoCredentialsError: Unable to locate credentials
[2023-03-03T07:44:55.989+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=metadata_scraper, task_id=scrape_nexrad_data, execution_date=20230303T074447, start_date=20230303T074455, end_date=20230303T074455
[2023-03-03T07:44:55.997+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 55 for task scrape_nexrad_data (Unable to locate credentials; 126)
[2023-03-03T07:44:56.048+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-03T07:44:56.067+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
